tests/test_dagbuilder.py

```diff
@@ -877,20 +888,6 @@ def test_make_dag_with_task_group_callbacks_tasks():
     # Can do something similar here as was done with the make_dag test (for Slack)
 
 
-def test_make_timetable():
```

Comment:
This test was removed because pre-commit identified two tests with the same name in this file and would not let me commit without it. If the reviewers prefer, I'm happy to create a separate PR for this.